### ISSUE TITLE
Work Package Detail Revamp - Overview Tab

### ISSUE THREAD

**Issue by RChandler234** [Northeastern-Electric-Racing/FinishLine#1122]
Blocked by #1121 

Populate the Overview tab with content (should have the same data as the details and Blocked By sections did on the old wp detail page, just without the pageblocks, see the project detail page or this PR: https://github.com/Northeastern-Electric-Racing/FinishLine/pull/1112, for an example of how to do that) - "Dependencies" should be replaced with "Blocked By" in the sc below:

This PR should merge into a feature branch called "#1097-wp-detail-revamp" instead of develop.

![image](https://user-images.githubusercontent.com/29521172/234751842-60727205-a356-40e6-8e23-96212af0269f.png)
Epic: #1097
